tests/func/experiments/conftest.py

```diff
@@ -102,7 +102,7 @@ def git_upstream(tmp_dir, erepo_dir):
 @pytest.fixture
 def git_downstream(tmp_dir, erepo_dir):
     url = "file://{}".format(tmp_dir.resolve().as_posix())
-    erepo_dir.scm.gitpython.repo.create_remote("upstream", url)
-    erepo_dir.remote = "upstream"
+    erepo_dir.scm.gitpython.repo.create_remote("downstream", url)
+    erepo_dir.remote = "downstream"
```

Comment:
This change shouldn't be necessary.

The `git_upstream` fixture creates an git repo that is "upstream" from tmp_dir. So the local `tmp_dir` git repo contains a remote named `upstream` that points to `erepo_dir`. This allows simulating that `tmp_dir` is a local repo, and `erepo_dir` is a server repo.

The `git_downstream` remote creates a repo that is downstream from `tmp_dir`. Meaning that it simulates `tmp_dir` being the "server" repo, and `repo_dir` being a downstream clone of `tmp_dir`. So the git repo in `tmp_dir` contains no remotes at all, and the git repo in `erepo_dir` contains a single remote named `upstream` which points to `tmp_dir`.